main.go

```diff
@@ -0,0 +1,35 @@
+package storageconsul
+
+import (
+	"github.com/caddyserver/caddy/v2"
+	"github.com/caddyserver/certmagic"
+)
+
+type TLSConsul struct {
+	storage *ConsulStorage
+}
+
+func init() {
+	caddy.RegisterModule(TLSConsul{})
+}
+
+func (TLSConsul) CaddyModule() caddy.ModuleInfo {
```

Comment:
```suggestion
func (tlsc *TLSConsul) CaddyModule() caddy.ModuleInfo {
```
(as mentioned above)

Reply:
(I'm mobile so can't see the whole thing)

This isn't needed I think? Is the variable ever used?

Reply:
Nope, it isn't. The reason for my suggestion was just consistency as explained above.
But looking at the code again, maybe it *can* really be used by returning it when `caddy.ModuleInfo` is created instead of creating a new instance.

Reply:
Noooo it needs to create a new instance, absolutely needs to be new.